.golangci.yml

```diff
@@ -185,3 +185,7 @@ issues:
     # that's a valid usage of bson.D
     - linters: [govet]
       text: "`go.mongodb.org/mongo-driver/bson/primitive.E` composite literal uses unkeyed fields"
+
+    # we don't want to introduce that constant
+    - linters: [goconst]
+      text: "string `_id` has 5 occurrences, make it a constant"
```

Comment:
"5" should be replaced with something. Does `text` support regular expressions?

Reply:
Looks like it supports regexp. Fixed it.